src/main/Tests/BattleshipModelTest.java

```diff
@@ -99,10 +99,23 @@ void fireAll() {
         for(int i=1;i<11;i++){
             for(int j=1;j<11;j++){
                 shot = new Coordinate(i,j);
-                testModel.fire(testModel.getAI(),shot);
+                testModel.fire(shot);
             }
         }
         assertNotEquals(testModel.getAI().getMisses().size(), 0);
     }
 
```

Comment:
Just changes to test based off of the changes to the player and AI firing functions. The tests for the easy static ship placement looks good too.